lib/connection.ex

```diff
@@ -116,15 +132,51 @@ defmodule Janus.Connection do
     end
   end
 
+  # janus gateway does not generate response for
+  # keep-alive messages so skip process of blocking caller and
+  # inserting transaction to pending table
   @impl true
+  def handle_call(
+        {:call, %{"janus" => "keepalive"} = payload, timeout},
+        _from,
+        state(
+          transport_module: transport_module,
+          transport_state: transport_state,
+          pending_calls_table: pending_calls_table
+        ) = state
+      ) do
+    transaction = generate_transaction!(pending_calls_table)
+
+    Logger.debug(
+      "[#{__MODULE__} #{inspect(self())}] Call: transaction = #{inspect(transaction)}, payload = #{
+        inspect(payload)
+      }"
+    )
+
+    payload_with_transaction = Map.put(payload, :transaction, transaction)
+
+    case transport_module.send(payload_with_transaction, timeout, transport_state) do
```

Comment:
Doesn't this create orphan transactions?
1. You send keep-alive
2. Do not insert transaction into transaction store
3. Ack will you from the server and will report as orphan transaction.

Reply:
Correct me if I am wrong though. Haven't tested this behaviour.

Reply:
Yup, it all seems to be true. I didn't now about `ack` response from `keepalive` messages.